### ISSUE TITLE
[Bug]Accident count by severity - missing TEXT field from API

### ISSUE THREAD

**Issue by yuvalbl** [hasadna/anyway-newsflash-infographics#186]
Currently Hard coded

**Screenshots**
![image](https://user-images.githubusercontent.com/11487010/80224779-dd82d280-8652-11ea-9cca-e65ae1468585.png)

**Comment by yuvalbl** [comment]
Currently all data returned as on long string (29/4)
Data should be broken into 4 frgaments (main content text, secondary text 1, secondary text 2, secondary text 3)
Should be implemented after API is updated

**Comment by yuvalbl** [comment]
```
export interface ILocationMeta {
  location_info: {
    resolution: string;
    road1: number;
    road_segment_name: string;
  };
  location_text: string;
};
```

**Comment by yuvalbl** [comment]
check out `accident_count_by_severity` in mock widgets data
data structure is a following:
```
data: {
        items: {
          severity_light_count: 81,
          severity_fatal_count: 1,
          severity_severe_count: 7,
          start_year: 2015,
          end_year: 2019,
          total_accidents_count: 89,
        },
      },
```

**Comment by SejoB** [comment]
All data comes from API out of text field
Should be implemented after API is updated

**Comment by yuvalbl** [comment]
partiality solved by #242
